setup.py

```diff
@@ -893,6 +917,10 @@ def get_all_provider_packages() -> str:
 class AirflowDistribution(Distribution):
     """The setuptools.Distribution subclass with Airflow specific behaviour"""
 
+    def __init__(self, attrs=None):
+        super().__init__(attrs)
+        self.install_requires = None
```

Comment:
What's this for?

Reply:
Fixing mypy error (not initialized `install_requires`). MyPy complains when local pre-commit is run.